src/components/OnboardDialog.vue

```diff
@@ -192,11 +334,49 @@ export default {
     closeDialog() {
       this.$emit("close");
     },
+    routeToDashboard() {
+      this.$router
+        .push({
+          name: "dashboard",
+          params: { contract: this.savedContractUuid }
+        })
+        .catch(() => {
+          log("*** Redirecting...");
+        });
+    },
+    async closeOnboarding() {
+      // TODO: Not sure if this is the best way to identify user made input on the contract form.
```

Comment:
Das ContractForm nutzt ([vuelidate](https://vuelidate-next.netlify.app/guide.html#the-dirty-state)), um zu prüfen, ob das Feld `$dirty` ist (wenn ich das richtig verstanden habe). Können wir das nutzen? Dann wäre es etwas strenger, weil unabhängig von der Gültigkeit der Eingabe. Aber ich glaube, das wäre der korrekte Ansatz, die Userinteraktion zu überwachen.